app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -82,6 +99,28 @@ protected function validator(array $data)
         ]);
     }
 
+    /**
+     * Get a validator for an incoming Manager registration request.
+     *
+     * @param  array $data Incoming registration data.
+     * @return \Illuminate\Contracts\Validation\Validator
+     */
+    protected function managerValidator(array $data)
```

Comment:
Question: When should you choose to abstract form validation rules into its own class?

Reply:
Uhhh, when the same rules are used in multiple places... so, like now...

Reply:
Okay, I extracted the validation rules. There's also a bit of duplicated code for initializing managers between the RegisterController and FirstVisitController, but I'm not sure where to put that.